src/main/java/trycb/config/Request.java

```diff
@@ -0,0 +1,19 @@
+package trycb.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.filter.CommonsRequestLoggingFilter;
+
+@Configuration
+public class Request {
+
+    public @Bean CommonsRequestLoggingFilter logFilter() {
```

Comment:
is this part related to User endpoint?

Reply:
It's just so we can see requests made to the backend in the logs, was quite hard to debug issues without.